src/Concerns/InteractsWithProperties.php

```diff
@@ -59,6 +59,12 @@ public function reset(...$properties)
                 $propertyName = $property->afterLast('.');
                 $objectName = $property->beforeLast('.');
 
+                if(method_exists($this->{$objectName}, 'reset'))
+                {
+                    $this->{$objectName}->reset($propertyName);
+                    continue;
+                }
```

Comment:
This seems unrelated to ->assertSet - please split different features into different PRs.

Also, please make the formatting match the rest of the code - your if statement formatting is different than the one above it. These small things make PRs harder to merge and make me more hesitant to merge them because if attention wasn't paid to details as superficial as formatting, it likely wouldn't be paid to more important factors.

Thanks!

Reply:
@calebporzio It is only related in that I encountered it while refactoring. Will move it to a new PR. Sorry about the formatting - am used to a slightly different style.